### ISSUE TITLE
v. 3.6.4 stops the OpenVINO plug-in from working! (with no warning!!_

### ISSUE THREAD

**Issue by kevintimba** [audacity/audacity#7396]
### Bug description

The OpenVINO AI separation is an absolute godsend. It was hard to get it running in 3.6.3 but once I did it began saving me many hours, to say nothing of a monthly Colab fee. I installed 3.6.4 immediately because it had such a long list of crashes it fixed but nowhere did it warn me that it would break Audacity's most import function. It was quite hard and time-consuming to get back to functional 3.6.3/OpenVino operation.

### Steps to reproduce

1. install 3.6.3 and OpenVINO
2. install 3.6.4

### Expected behavior

OpenVINO continues to work

### Actual behavior

OpenVINO stops working requiring many steps to fix.

### Audacity Version

latest stable version (from audacityteam.org/download)

### Operating system

Windows

**Comment by Kurtsley** [comment]
Have you downloaded the updated [OpenVINO plugin](https://github.com/intel/openvino-plugins-ai-audacity/releases) that was released alongside Audacity 3.6.4?

I ask this because I didn't seem to have an issue setting up the plugin with Audacity 3.6.4. Obviously I don't know what your exact setup is, but basic functions seemed to work on my Windows machine.

**Comment by kevintimba** [comment]
Thank you so much!!! (It would still be good if the 3.6.4 installer listed plug-ins that need upgrading but at least I'm good to go now.

**Comment by Kurtsley** [comment]
That may be tough to do, I've never looked at the installer though. I know you said if failed silently, but on my machine I received this message:

![image](https://github.com/user-attachments/assets/c47a02f0-4106-4a85-ad88-e8cfa5690f04)

While it doesn't explicitly tell you to update the mod, it lets you know that the mod isn't compatible with this version. I think it could be worded a bit better.  Maybe telling the user to manually update the mod?

**Comment by kevintimba** [comment]
I also got that message but now that I've been pointed
to audacity-win-v3.6.4-R3.4-64bit-OpenVINO-AI-Plugins.exe I installed
3.6.4, then got the message shown in your screenshot, then
ran audacity-win-v3.6.4-R3.4-64bit-OpenVINO-AI-Plugins.exe (quite a long
process - about 15 minutes) but it finally worked.

I'll probably do updates less often and always check for a new version of
OpenVINO before doing so, but in any case, it's fantastic to have the
highest quality AI separation within Audacity. I guess the best solution
would be to bundle it into the main installer but if someone didn't want
OpenVINO (hard to imagine) they might complain about the additional time.
